src/app/App.container.js

```diff
@@ -1,5 +1,5 @@
-import {connect} from 'react-redux';
-import {getSession, createSession, deleteSession} from 'state/contacts';
+import {connect} from '../../node_modules/react-redux';
```

Comment:
We shouldn't need to import directly from '../../node_modules', the build handles that for us. We should stick to importing just from the top level (e.g. `'react-redux')